tendrl/ceph_integration/sds_sync/__init__.py

```diff
@@ -609,9 +615,17 @@ def _request(self, method, obj_type, *args, **kwargs):
         request = getattr(request_factory, method)(*args, **kwargs)
 
         response = request.submit()
+
+        # Check the result status and accordingly set the error state
+        if response['status'] != 0:
```

Comment:
I have tested this, it is works fine this will solve the problems of
https://github.com/Tendrl/ceph-integration/issues/213
https://github.com/Tendrl/ceph-integration/issues/202